client/src/components/game/phases/PhaseContainer.vue

```diff
@@ -13,12 +13,12 @@
 
 <script lang="ts">
 import { Vue, Component } from 'vue-property-decorator';
-import ContainerInvestments from './ContainerInvestments.vue'
-import ContainerTrade from './ContainerTradeRework.vue';
-import ContainerPurchase from './ContainerPurchaseRework.vue';
-import ContainerDiscard from './ContainerDiscardRework.vue';
-import ContainerEvents from './ContainerEventsRework.vue';
-import ContainerDefault from './/ContainerDefault.vue';
+import ContainerInvestments from './investmentPhase/ContainerInvestments.vue'
```

Comment:
Number 1 can be changed, I don't _think_ there are any semantic differences.
Number 2 is done because they are broken up into the components that are used in the phase. We could move the containers into a single directory, but I thought it would be just as semantically accurate if every part that went into each phase was its respective phase file.

Reply:
I'd like to keep them in their own directory so that all of the components that have to to with X phase are in one location